conf.d/python.d/dockerd.conf

```diff
@@ -0,0 +1,79 @@
+# netdata python.d.plugin configuration for traefik health data API
+#
+# This file is in YaML format. Generally the format is:
+#
+# name: value
+#
+# There are 2 sections:
+#  - global variables
+#  - one or more JOBS
+#
+# JOBS allow you to collect values from multiple sources.
+# Each source will have its own set of charts.
+#
+# JOB parameters have to be indented (using spaces only, example below).
+
+# ----------------------------------------------------------------------
+# Global Variables
+# These variables set the defaults for all JOBs, however each JOB
+# may define its own, overriding the defaults.
+
+# update_every sets the default data collection frequency.
+# If unset, the python.d.plugin default is used.
+# update_every: 1
+
+# priority controls the order of charts at the netdata dashboard.
+# Lower numbers move the charts towards the top of the page.
+# If unset, the default for python.d.plugin is used.
+# priority: 60000
+
+# retries sets the number of retries to be made in case of failures.
+# If unset, the default for python.d.plugin is used.
+# Attempts to restore the service are made once every update_every
+# and only if the module has collected values in the past.
+# retries: 60
+
+# autodetection_retry sets the job re-check interval in seconds.
+# The job is not deleted if check fails.
+# Attempts to start the job are made once every autodetection_retry.
+# This feature is disabled by default.
+# autodetection_retry: 0
+
+# ----------------------------------------------------------------------
+# JOBS (data collection sources)
+#
+# The default JOBS share the same *name*. JOBS with the same name
+# are mutually exclusive. Only one of them will be allowed running at
+# any time. This allows autodetection to try several alternatives and
+# pick the one that works.
+#
+# Any number of jobs is supported.
+#
+# All python.d.plugin JOBS (for all its modules) support a set of
+# predefined parameters. These are:
+#
+# job_name:
+#     name: myname            # the JOB's name as it will appear at the
+#                             # dashboard (by default is the job_name)
+#                             # JOBs sharing a name are mutually exclusive
+#     update_every: 1         # the JOB's data collection frequency
+#     priority: 60000         # the JOB's order on the dashboard
+#     retries: 10             # the JOB's number of restoration attempts
+#     autodetection_retry: 0  # the JOB's re-check interval in seconds
+#
+# Additionally to the above, traefik plugin also supports the following:
+#
+#     url: '<scheme>://<host>:<port>/<health_page_api>'
+#     # http://localhost:8080/health
+#
+# if the URL is password protected, the following are supported:
+#
+#     user: 'username'
+#     pass: 'password'
+#
+# ----------------------------------------------------------------------
+# AUTO-DETECTION JOBS
+# only one of them will run (they have the same name)
+#
+local:
+    url: 'unix://var/run/docker.sock'
```

Comment:
I think we can have another one for the case netdata runs in a container itself:

```
local:
    url: 'unix://host/var/run/docker.sock'
```

So both at the same time. Only one will work.

Reply:
Hmmm I don't understand, a 2nd 
```
local:
    url: 'unix://host/var/run/docker.sock'
```

?

Reply:
I also don't understand that. @ktsaou what do you mean? Containerized netdata shouldn't have 'unix://host/var/run/docker.sock` socket, it is mounted as `'unix://var/run/docker.sock`

As per: https://github.com/firehol/netdata/wiki/Install-netdata-with-Docker#run-netdata-with-docker-command

Reply:
ok,

python plugin supports jobs. When 2 jobs have the same `name`, the code picks the first that works.

We use this feature for auto-detection. Let's assume nginx files on ubuntu are on `/var/log/www` and on centos on `/var/log/httpd`. We create 2 jobs with the same name, and the plugin will pick the first job that works. The same is true for example when we try to connect to a port. We try `localhost`, `127.0.0.1` and `::1` - only one will actually be used - the first that succeeds will obsolete the others.

Though the instructions I gave you above are wrong. The right config is:

```
local_run:
    name: 'local'
    url: 'unix://var/run/docker.sock'

local_host:
    name: 'local'
    url: 'unix://host/var/run/docker.sock'
```

So, the first of the above that will succeed, will obsolete the other. Only one job for each `name` is used.

Reply:
And now I saw that containerized netdata has the socket as `/var/run/docker.sock`, so the above are not needed.
Sorry...